Source/GSXib5KeyedUnarchiver.m

```diff
@@ -0,0 +1,3391 @@
+}
+
+- (id) decodeCellsForElement: (GSXibElement*)element
+{
+  id    object = nil;
+  Class class  = NSClassFromString([element attributeForKey: @"class"]);
+
+  if ([class isSubclassOfClass: [NSMatrix class]])
+    object = [self decodeFormCellsForElement: element];
+  else
+    object = [self decodeObjectForKey: @"cells"];
+
+  return object;
+}
+
+- (id) decodeNoAutoenablesItemsForElement: (GSXibElement*)element
+{
+  NSString  *autoenablesItems = [element attributeForKey: @"autoenablesItems"];
+  BOOL       value            = NO; // Default if not present...
+
+  if (autoenablesItems)
+    value = ![autoenablesItems boolValue];
+
+  return [NSNumber numberWithBool: value];
+}
+
+- (id) decodeMenuItemForElement: (GSXibElement*)element
+{
+  NSString      *itemID   = [element attributeForKey: @"selectedItem"];
+  GSXibElement  *itemElem = [objects objectForKey: itemID];
+
+  return [self objectForXib: itemElem];
+}
+
+- (id) decodeTitleCellForElement: (GSXibElement*)element
+{
+  NSString *title   = [element attributeForKey: @"title"];
+
+  if (title)
+    {
+      id   object  = [[NSCell alloc] initTextCell: title];
+      NSFont *font = [self decodeObjectForKey: @"titleFont"];
+
+      // IF no font...
+      if (font == nil) // default to system-11...
+        font = [NSFont systemFontOfSize: 11];
+
+      [object setAlignment: NSCenterTextAlignment];
+      [object setBordered: NO];
+      [object setEditable: NO];
+      [object setFont: font];
+      return AUTORELEASE(object);
+    }
+
+  return nil;
+}
+
+- (id) decodeBorderTypeForElement: (GSXibElement*)element
+{
+  NSString      *borderType = [element attributeForKey: @"borderType"];
+  NSBorderType   value      = NSGrooveBorder; // Cocoa default...
+
+  if (borderType)
+    {
+      if ([@"bezel" isEqualToString: borderType])
+        value = NSBezelBorder;
+      else if ([@"line" isEqualToString: borderType])
+        value = NSLineBorder;
+      else if ([@"none" isEqualToString: borderType])
+        value = NSNoBorder;
+      else
+        NSWarnMLog(@"unknown border type: %@", borderType);
+    }
+
+  return [NSNumber numberWithUnsignedInteger: value];
+}
+
+- (id) decodeModifierMaskForElement: (GSXibElement*)element
+{
+  id            object     = nil;
+  NSDictionary *attributes = [[element elementForKey: @"keyEquivalentModifierMask"] attributes];
+
+  // ??? SKIP modifier mask processing if BASE64-UTF8 string being used ???
+  if (attributes == nil)
+    {
+      if (([element elementForKey: @"keyEquivalent"]) &&
+          ([[element elementForKey: @"keyEquivalent"] attributeForKey: @"base64-UTF8"]))
+      {
+        object = [NSNumber numberWithUnsignedInt: 0];
+      }
+    else
+      {
+        // Seems that Apple decided to omit this attribute IF certain default keys alone
+        // are applied.  If this key is present WITH NO setting then the following is
+        // used for the modifier mask...
+        object = [NSNumber numberWithU
```

Comment:
Should this be commented out?

Reply:
Similar to above, there's some sort of a Github bug. This refers to lines:

```
#if 0
      // If a font is encoded then change the title to an attributed
      // string and set the font on it...
```

![image](https://user-images.githubusercontent.com/477596/71645876-9012dc00-2cd6-11ea-93eb-f93a045ac814.png)

Reply:
There are a few more places with "#if 0". I am really not sure what to do with these.